sections/literature_review/main.tex

```diff
@@ -0,0 +1,91 @@
+\section{Literature review}
+
+% Literature review
+A number of papers have been published that touch upon the use of 
+queueing models together with game theoretic concepts.
+In~\cite{FirmCompetition} the authors study a simultaneous price competition 
+between two firms that are modelled as two distinct queueing systems with a 
+fixed capacity and a combined arrival rate.
+They calculate the Nash equilibrium both for identical and heterogeneous firms
+and show that for the former a pure Nash equilibrium always exist and for the 
+latter a unique equilibrium exists where only one firm operates.
+The authors have also extended their model in~\cite{FirmCompetition2} by 
+allowing the players (firms) to choose capacities. 
+A main result from this paper was that when both firms operate independently as
+a monopoly, the equilibria are socially optimal, but this is not the case when
+the firms operate together.
+Another extension of~\cite{FirmCompetition} was introduced 
+in~\cite{FirmCompetitionExtension} where a long-run version of the competition 
+was considered that also had capacity as a decision variable.
+In~\cite{knight2017measuring} a normal form game is built that is informed by a 
+two-dimensional Markov chain in order to model interactions between critical
+care units.
```

Comment:
This feels out of place. Should be lower?

Reply:
Yeah for sure. Will move it in the second paragraph